docs/deployment-process/execution-containers-for-workers/index.md

```diff
@@ -38,10 +38,11 @@ The same process can be repeated for a runbook.
 Pre-pulling your chosen image will save you time during deployments.
 :::
 
-Your first deployment in a Docker container might take a while as it will download all the dependencies on the docker image. These dependencies will be installed as a part of deployment process execution automatically. Alternatively, you can pre-pull the desired docker image before your first deployment to avoid any delays.
+When you choose to run one or more of your deployment steps in a container, your deployment process will `docker pull` the image you provide at the start of each deployment during package acquisition.
 
-## What docker image should I use?
+For your first deployment this may take a while since your docker image won't be cached. You can pre-pull the desired docker image on your worker before your first deployment to avoid any delays.
```

Comment:
```suggestion
For your first deployment, this may take some time because your docker image isn't cached. You can pre-pull the desired docker image on your worker before your first deployment to avoid delays.
```